src/mbgl/style/style_layer.hpp

```diff
@@ -45,6 +45,8 @@ class StyleLayer {
     // Returns true if any paint properties have active transitions.
     virtual bool recalculate(const StyleCalculationParameters&) = 0;
 
+    virtual bool isHidden() const = 0;
```

Comment:
Rename this to `isVisible` and reverse the boolean, for consistency with "visibility" as from the style spec.